microsoft-edge/devtools-guide-chromium/whats-new/2021/10/devtools.md

```diff
@@ -0,0 +1,98 @@
+
+
+<!-- ====================================================================== -->
+## Improved keyboard access for defining User-Agent Client Hints
+
+<!-- Title: Improved keyboard access when navigating to User agent client hints in Settings -->
+<!-- Subtitle: When adding a custom device to emulate in DevTools, you can now expand the User agent client hints section more easily. -->
+
+Starting with [Microsoft Edge version 92](../05/devtools.md#user-agent-client-hints-for-devices-in-the-network-conditions-tab), you can specify User-Agent Client Hints when defining the user agent string in the **Network conditions** tool or when adding a custom device to emulate in **Settings**.
+
+In earlier versions of Microsoft Edge, when adding a custom device to emulate from the **Devices** section of **Settings**, selecting the **User agent client hints** button with the keyboard caused the incorrect UI item to be activated.  Instead of expanding the **User agent client hints** section for you to fill it in, it selected the **Add** button.  The device you were starting to define was immediately added, using empty fields in the **User agent client hints** section.
+
+In Microsoft Edge version 95, this issue has been fixed.  Selecting the **User agent client hints** button by using the keyboard now expands a form, in which you can specify client hints for the custom device.
+
+To learn more about User-Agent Client Hints, see [Detecting Microsoft Edge from your website](../../../../web-platform/user-agent-guidance.md#user-agent-client-hints).
+
+:::image type="content" source="../../media/2021/10/keyboard-define-ua-client-hints.png" alt-text="Description" lightbox="../../media/2021/10/keyboard-define-ua-client-hints.png":::
+
+To view the history of this feature in the Chromium open-source project, see [Issue 1243827: User agent client hints form accessibility](https://bugs.chromium.org/p/chromium/issues/detail?id=1243827).
+
+
+<!-- ====================================================================== -->
+## Console filters now display grouped messages if the filter matches the group title
+
+<!-- Title: Improved filtering for grouped messages in the Console -->
+<!-- Subtitle: Filters in the Console is now more intuitive, displaying grouped messages only when the filter matches the group label. -->
+
+In earlier versions of Microsoft Edge, there were several problems when trying to filter grouped messages in the Console. You can use `console.group()` to label a group of messages and provide some organization for Console messages. However, when filtering to a group label, the Console wouldn't display individual messages inside the group.  Also, filtering wouldn't hide group labels, even if the label didn't match the filter.
```

Comment:
```suggestion
You can use `console.group()` to label a group of messages and provide some organization for Console messages. In earlier versions of Microsoft Edge, there were several problems when trying to filter grouped messages in the Console. When filtering to a group label, the Console wouldn't display individual messages inside the group.  Also, filtering wouldn't hide group labels, even if the label didn't match the filter.
```
I just moved the second sentence to the first one here just because it made more sense to me to say: here's how you can do x. There are several problems with filtering when doing x. Here are the problems.